public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -229,7 +229,7 @@ block get-heroes-details
   ## Update hero details
 
   We can edit a hero's name already in the hero detail view. Go ahead and try
-  it. As we type, the hero name is updated in the view heading. 
+  it. As we type, the hero name is updated in the view heading.
```

Comment:
@wardbell mentioned to me a few days ago that we are switching from the "we" to the "you" in our prose. If so, this needs to switch. But check with Ward.